packages/dataviews/src/dataform-controls/datetime.tsx

```diff
@@ -34,7 +33,7 @@ export default function DateTime< Item >( {
 				<VisuallyHidden as="legend">{ label }</VisuallyHidden>
 			) }
 			<TimePicker
-				currentTime={ value }
+				currentTime={ typeof value === 'symbol' ? '' : value }
```

Comment:
I don't see the check about symbol in other controls like integer, why?

Reply:
I will update it, the TimePicker component it-self seemed to handle it gracefully, but its not expected.